src/Service/API/ShipmentApiService.php

```diff
@@ -190,4 +194,87 @@ private function isTrackingEmailAllowed()
     {
         return (bool) \Configuration::get(Config::SEND_EMAIL_ON_PARCEL_CREATION);
     }
+
+    private function trimRemarkMessage($message) {
+        return strlen($message) > self::REMARK_LIMIT ? substr($message,0,self::REMARK_LIMIT)."..." : $message;
+    }
+
+    private function convertAccentsAndSpecialToNormal($string) {
```

Comment:
First of all, this should be moved to string utility, somewhere outside class, secondly perhaps there is more clean way to do the job? Third, we are going to need unit tests to cover this functionality.

Reply:
I moved functionality to string Utilities. There is one more way using 3rd party library Latinidze, but not sure how much cleaner it will be, because it's JS library, I will need to make webpack and package files + to build js bundle for it and than use Latinidze class. About Unit test, I will make research since never wrote them. Can I get more detailed instructions where I should write them i see tests/Unit folder but not sure where to start.